js/app.jsx

```diff
@@ -412,11 +437,13 @@ var AddToSuite = React.createClass({
         <SearchableCaseSelectionList isNotIn={true} 
                                      suiteId={this.state.suite.id}
                                      onCheck={this.handleAdd}
+				     isCaseListUpdate={this.state.isCaseListUpdate}
```

Comment:
Agreed. Now, I load remote suite list at "componentWillReceiveProps" state of "SearchableCaseSelectionList". However, it would send lots of unnecessary request...